services/brig/test/integration/API/Provider.hs

```diff
@@ -419,202 +427,61 @@ testListServices config db brig = do
     -- (and repeatedly) against a shared database and thus a shared
     -- "name index" per tag.
     uniq <- UUID.toText . toUUID <$> randomId
```

Comment:
`testListServices` went through some refactoring because a lot of code here is shared with `testSearchWhitelist`.